src/FixedDataTableBufferedRows.js

```diff
@@ -50,7 +50,6 @@ var FixedDataTableBufferedRows = React.createClass({
   },
 
   componentDidMount() {
-    setTimeout(this._updateBuffer, 1000);
```

Comment:
I'm hoping this wasn't needed, as _updateBuffer was not defined. Need some input here.

Before I removed this, >npm run test failed most FDTRoot-test tests with TypeError: "callback" argument must be a function, whereas they all passed with >npm run test:server. This is the difference I am referring to in my README.md changes.

Any idea what is going on there, or why we are seeing this difference?